Projects/Modules/Documentarian/Source/Public/Functions/General/Get-ContentWithoutHeader.ps1

```diff
@@ -2,7 +2,6 @@
 # Licensed under the MIT License.
 
 function Get-ContentWithoutHeader {
-
     [CmdletBinding()]
```

Comment:
Nit: I _think_ this function is implicitly returning an array of strings, should we make this explicit?


```suggestion
    [CmdletBinding()]
    [OutputType([string[]])]
```